### PR TITLE
refactor(components): el-message-box icon

### PR DESCRIPTION
**Pull request by adaex**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/message-box/__tests__/message-box.spec.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { mount } from '@vue/test-utils'
 import { rAF } from '@element-plus/test-utils/tick'
 import { triggerNativeCompositeClick } from '@element-plus/test-utils/composite-click'
+import { QuestionFilled } from '@element-plus/icons'
+import { TypeComponentsMap } from '@element-plus/utils/icon'
```
Comment (sxzz) [inline packages/components/message-box/__tests__/message-box.spec.ts:5]:
```
'TypeComponentsMap' is defined but never used
```

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
@adaex Same as this one, we should use `16px` instead of the default `14px`

**Comment by imswk** [comment]
image
The color of the icon is invalid

**Comment by imswk** [comment]
image
Message component  color of the icon is invalid

**Comment by imswk** [comment]
https://github.com/element-plus/element-plus/blob/55348b30b6ae59b0efe8abfa9f8dbbeaabb5614a/packages/components/message/src/types.ts#L12

Message component  `iconClass` is changed to `icon` and the type is changed to `string | Component`

**Comment by adaex** [comment]
> @adaex Same as this one, we should use `16px` instead of the default `14px`

done.

**Comment by adaex** [comment]
> Message component color of the icon is invalid

solved.

**Comment by adaex** [comment]
> Seems like it got conflicts...

😖😖😖

**Comment by imswk** [comment]
LGMT

**Pull request merged**